test_cases/scripts/unit_test_non_blocking_sleep.py

```
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Ruff (0.13.3)

Shebang is present but file is not executable

(EXE001)